### PR TITLE
Allows user to update card details

### LINKED ISSUE #1241: Enable premium users to update card details
As the number of premium users increases this is going to become more and more important.  Some related links:

* https://stripe.com/docs/recipes/updating-customer-cards
* http://stackoverflow.com/questions/20065939/change-credit-card-information-stripe

### PR DESCRIPTION
**Pull request by tansaku**
fixes #1241

### CODE REVIEW

File: app/views/charges/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<%= form_tag charge_path('tansaku'), method: :put do %>
```
Comment (tansaku) [inline app/views/charges/edit.html.erb]: @mtc2013 we should probably remove this hard coding

File: app/views/charges/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  data-key="<%= Rails.configuration.stripe[:publishable_key] %>"
+  data-name="Agile Ventures"
+  data-panel-label="Update Card Details"
+  data-label="Update Card Details"
+  data-allow-remember-me=false
+  data-locale="auto">
+  </script>
+<% end %>
+
+<!--data-image="/path/to/your/logo.png"-->
```
Comment (tansaku) [inline app/views/charges/edit.html.erb]: should remove this or get images sorted

**Comment by tansaku** [comment]
hmm, and I was wondering if we needed to make the cukes more imperative ...

**Comment by diraulo** [comment]
@tansaku is this ready to go? I need to update my card number 😉

**Comment by tansaku** [comment]
@diraulo yes, this is good to deploy - we will have to do a little data migration before you can use it though

**Pull request merged**